config/jobs/kubernetes/kops/templates/periodic-scenario.yaml.jinja

```diff
@@ -5,9 +5,6 @@
   labels:
     preset-service-account: "true"
     preset-aws-ssh: "true"
-    {%- if not boskos_resource_type %}
-    preset-aws-credential: "true"
-    {%- endif %}
```

Comment:
This is impact the scale tests because the credentials are needed in another cluster.